tests/meshcop/meshcop

```diff
@@ -458,6 +436,28 @@ ba_start()
   write_syslog "AGENT: start complete"
 }
 
+web_start()
+{
+  write_syslog "WEB: kill old"
+  sudo killall "${OTBR_WEB}" || true
+  write_syslog "WEB: starting"
+  (cd "${ORIGIN_PWD}" && sudo "${OTBR_WEB_PATH}" -I "${TUN_NAME}"&)
+  sleep 10
+
+  pidof ${OTBR_WEB} || die "WEB: failed to start"
+  write_syslog "WEB: start complete"
+}
+
+network_form()
+{
+  readonly OT_PANID="$(random_panid)"
+  readonly OT_XPANID="$(random_xpanid)"
+  readonly OT_MASTER_KEY="$(random_masterkey)"
+  readonly OT_CHANNEL="$(random_channel)"
+
+  curl --header "Content-Type: application/json" --request POST --data "{\"networkKey\":\"${OT_MASTER_KEY}\",\"prefix\":\"fd11:22::\",\"defaultRoute\":true,\"extPanId\":\"${OT_XPANID}\",\"panId\":\"${OT_PANID}\",\"passphrase\":\"${OT_AGENT_PASSPHRASE}\",\"channel\":${OT_CHANNEL},\"networkName\":\"${OT_NETWORK_NAME}\"}" http://127.0.0.1/form_network || die "WEB: foram failed"
```

Comment:
use single quote to avoid those escapes `\"`?

Reply:
In single quote, `${xxx}` will be parsed as raw string.